From 8c896a0d7e4d366c54badf760b0c737242a88d53 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:04 -0400
Subject: [redhat] kabi: Adding symbol __const_udelay (arch/x86/um/delay.c)

Message-id: <8ee1a0b6237afdbf965af95283140378842027e8.1410510773.git.poros@redhat.com>
Patchwork-id: 96048
O-Subject: [PATCH RHEL7.1 004/233] kabi: Adding symbol __const_udelay (arch/x86/um/delay.c)
Bugzilla: 1071097
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    Called via mdelay to retry LLT send operation.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/__const_udelay

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_x86_64/__const_udelay b/redhat/kabi/kabi-module/kabi_x86_64/__const_udelay
new file mode 100644
index 0000000..d376cfd
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/__const_udelay
@@ -0,0 +1,2 @@
+#1-
+0xeae3dfd6 __const_udelay vmlinux EXPORT_SYMBOL
-- 
1.7.1